android/src/main/java/co/touchlab/droidcon/android/MainActivity.kt

```diff
@@ -71,6 +74,17 @@ class MainActivity : ComponentActivity(), KoinComponent {
                 email = user.email,
                 pictureUrl = user.photoUrl?.toString(),
             )
+            if (!ChatManager.isConnected) {
+                ChatManager.initChat(
+                    applicationContext = applicationContext,
+                    userData = UserData(
+                        id = user.uid,
+                        name = user.displayName,
+                        email = user.email,
+                        pictureUrl = user.photoUrl?.toString(),
+                    ),
+                )
+            }
```

Comment:
Is the chat only enabled for authenticated people? If yes, should we create a placeholder screen for the chat tab?

Reason: if I'm not authenticated, the app crashes

Reply:
This logic does not update the current user when switching accounts.